src/aat/resources/features/F-037/F-037.feature

```diff
@@ -0,0 +1,68 @@
+@F-037
+Feature: F-037: Submit event for an existing case (V2)
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-024
+  Scenario: should create event successfully for an existing case
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a successful call [to get an event token for the case just created] as in [S-024-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a case Id for just created case]
```

Comment:
should provide the accurate info for full visibility is not very much clear i.e. 
And the request [contains a case that has just been created as in Standard_Full_Case_Creation_Data]

Reply:
This step is part of same scenario and there is only one case created in above step, So isn't that obvious? Moreover the request only contains the case id passed as path variable so your suggested spec is not making enough sense to me.

Reply:
updated to 'contains a case Id that has just been created as in Standard_Full_Case_Creation_Data'